specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json

```diff
@@ -1354,6 +1354,32 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "identity": {
```

Comment:
@ravbhatnagar should this definition match https://github.com/Azure/azure-rest-api-specs-pr/blob/current/specification/common-types/resource-management/v1/types.json#L200? I see the enum as modelAsString: true here, but false in the "common-types". 
Also, could we match the descriptions?

Reply:
@veronicagg we should be consistent with compute:
https://github.com/Azure/azure-rest-api-specs/blob/current/specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json#L4709-L4734

Consistency here will simplify a lot samples at the SDK level

Reply:
Fixed